packages/@aws-cdk/aws-stepfunctions/lib/step-functions-task.ts

```diff
@@ -78,3 +78,28 @@ export interface StepFunctionsTaskConfig {
    */
   readonly metricDimensions?: cloudwatch.DimensionHash;
 }
+
+/**
+ * Ways to call an integrated AWS service in Step Functions
+ *
+ * @default REQUEST_RESPONSE
+ *
+ * @see
+ * https://docs.aws.amazon.com/step-functions/latest/dg/connect-to-resource.html
+ */
+export enum ServiceIntegrationPattern {
+  /**
+   * Call a service and let Step Functions progress to the next state immediately after it gets an HTTP response.
+   */
+  REQUEST_RESPONSE = '',
```

Comment:
I renamed `REQUEST_RESPONSE` as `FIRE_AND_FORGET` and `RUN_A_JOB` as `SYNC`.

Reply:
For clarity, I'm not mandating: I'm only trying to engage you in discussion here. Since I believe "request response" is the terminology used in your official documentation, I'm not sure it's worth changing the term since it may confuse users.

If you as a service owner are okay with deviating from the terminology as a conscious choice, I would welcome the change. If the terminology should stay the same, I completely understand, but then we should make the behavior very clear in the doc string.